scripts/irods/test/test_iput.py

```diff
@@ -928,6 +926,39 @@ def test_server_honors_server_config_option_checksum_read_buffer_size_in_bytes__
             # self.object_name deleted in teardown
             os.unlink(local_file)
 
+    def test_server_rejects_invalid_server_config_options_for_checksum_read_buffer_size_in_bytes__issue_7947(self):
+        self.object_name = 'test_iput_with_updated_checksum_read_buffer_size_in_bytes__issue_7947'
```

Comment:
This doesn't matter too much, but it seems the object name still reflects the old test name.

Reply:
done